### ISSUE TITLE
[QST] Python Conv2d example?

### ISSUE THREAD

**Issue by akirchmeyer** [NVIDIA/cutlass#474]
Hello, I would like to implement a custom Pytorch kernel using the CUTLASS 2D convolution. I saw that you released Python scripts in release 2.9 to launch a GEMM kernel from Python. Is there a similar script to launch a 2D convolution fprop kernel? Same for wgrad and dgrad? I am currently looking at how to transpose what you did for GEMM but I was wondering whether you had already done the work. That would be very helpful. Thanks a lot!

**Comment by kerrmudgeon** [comment]
Hello! Thank you for expressing interest.

There is not a similar script for convolution, but there could be. Nearly all of the details are implemented already or follow the existing pattern. I expect it would be low effort.

Could I work with you to add fprop initially? I can devote some time this week and submit a pull request to you for evaluation. 

Once that works and you have had the opportunity to try it out, perhaps you would be willing to add additional steps for to complete the convolution gradient passes? :)

**Comment by akirchmeyer** [comment]
Hi Andrew,
Yes that would be great, thank you!
Sure no problem, I could try to complete the gradient versions once fprop is done, this way I'll see if I understood the design or not.

**Comment by hwu36** [comment]
it is supported in 2.10
